apiserver/common/action.go

```diff
@@ -166,14 +170,14 @@ func Actions(args params.Entities, actionFn func(string) (state.Action, error))
 // It needs a tagToActionReceiver function and a registerFunc to register
 // resources.
 // It's a helper function currently used by the uniter and by machineactions
-func WatchOneActionReceiverNotifications(tagToActionReceiver func(tag string) (state.ActionReceiver, error), registerFunc func(r facade.Resource) string) func(names.Tag) (params.StringsWatchResult, error) {
+func WatchOneActionReceiverNotifications(watchChanges bool, tagToActionReceiver func(tag string) (state.ActionReceiver, error), registerFunc func(r facade.Resource) string) func(names.Tag) (params.StringsWatchResult, error) {
```

Comment:
This is confusing at first read cause it seems we are creating a watcher but optionally not watching for changes.
I think this may be better split up as 2 api calls:
- WatchPendingActionsForReceiver
- WatchOneActionReceiverNotifications

These would also be on the receiver below etc